java/src/main/java/io/cucumber/gherkin/GherkinLine.java

```diff
@@ -78,26 +77,52 @@ List<GherkinLineSpan> parseTags() {
         if (empty) {
             return emptyList();
         }
-        String uncommentedLine = StringUtils.removeComments(text);
-        int indexInUncommentedLine = 0;
 
-        String[] elements = uncommentedLine.split(TAG_PREFIX);
-        if (elements.length == 0) {
+        // look for the first tag
+        int indexStartTag = text.indexOf(TAG_PREFIX_CHAR);
```

Comment:
Because there is a next tag too, please call this `indexStartCurrentTag`. This signals in advance that there will be pointers to other tags too.